### ISSUE TITLE
Indicate which parts of the output is different from the test case

### ISSUE THREAD

**Issue by karlr-dev** [JuiceMind/JuiceMind#414]
Could indicate differences with red underlining:

![Image](https://github.com/user-attachments/assets/ce5624e3-7a8b-4c35-91b2-544b54a22b70)

**Comment by teach8** [comment]
Yes, this is crucial. Students are spending way too long trying to figure out why tests are failing.

**Comment by juicemind-dev** [comment]
Closing because duplicate of this: https://github.com/JuiceMind/JuiceMind/issues/321
